app/server_test.go

```diff
@@ -201,3 +205,63 @@ func checkEndpoint(t *testing.T, client *http.Client, url string, expectedStatus
 
 	return nil
 }
+
+func TestPanicLog(t *testing.T) {
+	// Creating a temp file to collect logs
+	tmpfile, err := ioutil.TempFile("", "mlog")
+	if err != nil {
+		require.NoError(t, err)
+	}
+
+	defer os.Remove(tmpfile.Name())
```

Comment:
Please verify that the temp file is indeed deleted (check the error, t.Fatal if not?). I think you need to tmpfile.Close() before `os.Remove()` succeeds, but either way I'd say check the os.Remove() result.